azurelinuxagent/common/event.py

```diff
@@ -271,37 +270,34 @@ def is_period_elapsed(self, delta, h):
             (self.periodic_events[h] + delta) <= datetime.now()
 
     def add_periodic(self, delta, name, op=WALAEventOperation.Unknown, is_success=True, duration=0,
-                     version=str(CURRENT_VERSION), message="", evt_type="", is_internal=False, log_event=True,
-                     force=False):
+                     version=str(CURRENT_VERSION), message="", log_event=True, force=False):
         h = hash(name + op + ustr(is_success) + message)
 
         if force or self.is_period_elapsed(delta, h):
             self.add_event(name, op=op, is_success=is_success, duration=duration,
-                           version=version, message=message, evt_type=evt_type,
-                           is_internal=is_internal, log_event=log_event)
+                           version=version, message=message, log_event=log_event)
             self.periodic_events[h] = datetime.now()
 
     def add_event(self, name, op=WALAEventOperation.Unknown, is_success=True, duration=0, version=str(CURRENT_VERSION),
-                  message="", evt_type="", is_internal=False, log_event=True):
+                  message="", log_event=True):
 
         if (not is_success) and log_event:
             _log_event(name, op, message, duration, is_success=is_success)
 
-        self._add_event(duration, evt_type, is_internal, is_success, message, name, op, version, event_id=1)
+        self._add_event(duration, is_success, message, name, op, version, event_id=1)
 
-    def _add_event(self, duration, evt_type, is_internal, is_success, message, name, op, version, event_id):
+    def _add_event(self, duration, is_success, message, name, op, version, event_id):
```

Comment:
I removed the `evt_type` and `is_internal` parameters because they always defaulted to `""` and `False`, respectively. Furthermore, these two values are for fields `ExtensionType` and `IsInternal` which are common to agent and extension events, so I had to refactor them out of this method and instead populate them in `EventLogger.add_common_parameters_to_event`.